diff -ur ORBit2-2.14.19.orig/configure.in ORBit2-2.14.19/configure.in
--- ORBit2-2.14.19.orig/configure.in	2010-07-01 13:57:01.000000000 +0000
+++ ORBit2-2.14.19/configure.in	2013-04-01 20:41:55.902571944 +0000
@@ -262,7 +262,7 @@
 
 if test "$enable_maintainer_mode" = "yes"; then
 	DISABLE_DEPRECATED_CFLAGS=" \
-		-DG_DISABLE_DEPRECATED -DG_DISABLE_SINGLE_INCLUDES"
+		-DG_DISABLE_SINGLE_INCLUDES"
 	AC_SUBST(DISABLE_DEPRECATED_CFLAGS)
 fi
 
diff -ur ORBit2-2.14.19.orig/linc2/src/Makefile.am ORBit2-2.14.19/linc2/src/Makefile.am
--- ORBit2-2.14.19.orig/linc2/src/Makefile.am	2010-02-09 12:05:35.000000000 +0000
+++ ORBit2-2.14.19/linc2/src/Makefile.am	2013-04-01 20:42:37.883571886 +0000
@@ -12,8 +12,7 @@
 	-I$(top_builddir)/linc2/include  \
 	-I$(top_srcdir)/linc2/include    \
 	$(LINC_CFLAGS)             \
-	$(WARN_CFLAGS)             \
-	-DG_DISABLE_DEPRECATED
+	$(WARN_CFLAGS)             
 
 if OS_WIN32
 else
--- ORBit2-2.14.19/test/everything/Makefile.am~	2013-12-07 10:59:17.000000000 +0100
+++ ORBit2-2.14.19/test/everything/Makefile.am	2013-12-07 11:00:27.439835598 +0100
@@ -78,7 +78,7 @@
 BUILT_SOURCES = ${EVERYTHING_BUILT} everything-imodule.c
 CLEANFILES = ${BUILT_SOURCES} iorfile
 
-TESTS = $(srcdir)/test.sh
+TESTS = test.sh
 
 EXTRA_DIST = $(IDL_FILES) test.sh $(included_src)
 
diff -urN ORBit2-2.14.19.orig/src/services/name/Makefile.am ORBit2-2.14.19/src/services/name/Makefile.am
--- ORBit2-2.14.19.orig/src/services/name/Makefile.am	2019-03-15 10:25:22.576738108 +0100
+++ ORBit2-2.14.19/src/services/name/Makefile.am	2019-03-15 10:27:17.775898401 +0100
@@ -68,7 +68,7 @@
 
 orbit_name_server_2_SOURCES = boot.c
 orbit_name_server_2_LDFLAGS = $(FLAGS)
-orbit_name_server_2_DEPENDENCIES = $(DEPS) CosNaming.h
+orbit_name_server_2_DEPENDENCIES = $(DEPS) CosNaming.h libname-server-2.a
 orbit_name_server_2_LDADD = libname-server-2.a $(LDADDS)
 
 $(libname_server_2_a_OBJECTS) $(name_client_2_OBJECTS) $(orbit_name_server_2_OBJECTS) : CosNaming.h
